lib/src/main/java/graphql/nadel/validation/NadelUnionValidation.kt

```diff
@@ -48,13 +46,7 @@ internal class NadelUnionValidation(
                 extraType = memberOverallType as GraphQLObjectType,
             )
         } else {
-            typeValidation.validate(
-                NadelServiceSchemaElement.from(
-                    service = parent.service,
-                    overall = memberOverallType,
-                    underlying = memberUnderlyingType,
-                ),
-            )
```

Comment:
Same as other comments, this is actually already validated. So this was a useless call.